<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1541</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1521</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>831</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="glEmbedHook"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QDeclarativeView" name="qmlEmbedHook">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1500</width>
        <height>750</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="Dense2Pattern">
        <color alpha="255">
         <red>150</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="source">
       <url>
        <string/>
       </url>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTextEdit" name="ui_output">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1021</width>
        <height>801</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1541</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDeclarativeView</class>
   <extends>QGraphicsView</extends>
   <header>QtDeclarative/QDeclarativeView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
